### ISSUE TITLE
Should line 14 of cell_filter.js say 'jquery' instead of 'jqueryui'?

### ISSUE THREAD

**Issue by mister-average** [ipython-contrib/jupyter_contrib_nbextensions#1397]
Cell filter works for me only if I make that change.  Otherwise I get '$ is not a function'.

**Comment by jcb91** [comment]
Yes, it probably should. I don't think it uses any jqueryui-specific methods. Feel free to open a PR

**Comment by aditya95sriram** [comment]
I don't think this has been fixed right ? In my browser's developer console I see the error "`$ is not a function`" which makes the case stronger for this to be jQuery. 

I can open a PR if that's all the changes required.

**Comment by aditya95sriram** [comment]
This hasn't been fixed yet, right ? I'm still getting the error "`$ is not a function`".
